### PR TITLE
fix(nemesis.py): filter out schema_error during init node

### PR DESCRIPTION
**Pull request by fgelcer**
seen during lwt parallel schema changes some issues
on schema sync during init nodes, hence lowering the
priority to not fail the tests.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -761,18 +761,23 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, timeout=HOUR_IN_SEC *
...
-        try:
-            self.cluster.wait_for_init(node_list=[new_node], timeout=timeout, check_node_health=False)
-            self.cluster.clean_replacement_node_ip(new_node)
-        except (NodeSetupFailed, NodeSetupTimeout):
-            self.log.warning("TestConfig of the '%s' failed, removing it from list of nodes" % new_node)
-            self.cluster.nodes.remove(new_node)
-            self.log.warning("Node will not be terminated. Please terminate manually!!!")
-            raise
-        self.cluster.wait_for_nodes_up_and_normal(nodes=[new_node])
+        with EventsSeverityChangerFilter(new_severity=Severity.WARNING,
```
Comment (roydahan) [inline sdcm/nemesis.py:764]: I'm not sure this is something we want.
If node joins and somehow it fails to load schema, we want see this error.

Comment (fgelcer) [reply sdcm/nemesis.py:764]: the idea was to reduce the severity of the issue and not just ignore it

**Comment by bentsi** [comment]
Link to the Trello task is missing from description

**Comment by fgelcer** [comment]
> Link to the Trello task is missing from description

@bentsi , this PR was sent as part of investigation of a 4.5 job, and therefor no Trello task was created for it

File: sdcm/nemesis.py (hunk 1 above)
Comment (roydahan) [reply sdcm/nemesis.py:764]: But we ignore warnings most of  the time and we call it SUCCESS.
So, this is not a good enough solution, it's too risky to miss issues.

Comment (fgelcer) [reply sdcm/nemesis.py:764]: in the case of a real issue, won't this schema mismatch continue to appear after the node joined the cluster?
what would you suggest?

Comment (roydahan) [reply sdcm/nemesis.py:764]: I don't know.
We should get an answer from Shlomi or a developer who knows this area if this is good enough to ignore schema mismatch during any node init.

**Comment by roydahan** [comment]
@fgelcer what's the status of this one?
You had an issue in Scylla which I recall isn't a real issue but SCT issue with filtering.
(Please add the link here).
In that case, this PR isn't needed and should be closed.

**Comment by fgelcer** [comment]
> @fgelcer what's the status of this one? You had an issue in Scylla which I recall isn't a real issue but SCT issue with filtering. (Please add the link here). In that case, this PR isn't needed and should be closed.

@roydahan , IIRC this is the issue related to it --> https://github.com/scylladb/scylla/issues/4648
and it is an issue, so this PR will do avoid to fail the test...

**Comment by roydahan** [comment]
No it's not the issue and I remove the link so it won't reference irrelevant issues.

**Pull request closed without merging**